VBA/Office-Shared-VBA/articles/about-autosave.md

```diff
@@ -0,0 +1,120 @@
+---
+title: About AutoSave in Office
+ms.prod: MULTIPLEPRODUCTS
+ms.assetid: about-autosave
+---
+
+
+# About AutoSave in Office
+
+Learn about how AutoSave works in Excel, PowerPoint, and Word and how it can impact add-in or macro integration. For more about how AutoSave works in general, see [this link](https://support.office.com/en-US/article/What-is-AutoSave-6d6bd723-ebfd-4e40-b5f6-ae6e8088f7a5).
+
+## Overview of AutoSave
+
+When a file is hosted in the cloud (that is, OneDrive, OneDrive for Business, or SharePoint Online), AutoSave enables the user's edits to be saved automatically and continuously. When the file is shared with others, then the others' changes will be merged into this user's version of the file. If AutoSave is turned off, then save must be triggered manually for the user's changes to be persisted in the cloud and for that user to receive others' changes.
+
+Currently, Excel, Word, and PowerPoint provide a **BeforeSave** event that allows a developer to execute code after the user triggers a save but before the save actually occurs. Excel also provides an **AfterSave** event that can execute macro or add-in code after the save completes.
+
+When AutoSave is enabled, these events will fire automatically on a periodic basis without user interaction. Because of this new behavior when AutoSave is enabled, add-ins and macros that leverage these events may experience problems when AutoSave is on.
+
+In general, these issues can be avoided if the user chooses to disable AutoSave. You can also do this on the user’s behalf using the **autoSaveOn** property (see the example below). You can also take steps as a developer to mitigate these potential problems so that your add-ins and macros work smoothly, even if AutoSave is enabled.
+
+## Potential issues with save events and AutoSave
```

Comment:
Unlike the other article, where each section only had 1-2 examples, the list here (5 examples) is sufficiently long that a summary table might be helpful. What do you think?